pkg/config/lifecycle_provider.go

```diff
@@ -58,17 +66,17 @@ func (l *LifecycleProvider) LayerForOS(os string) (v1.Layer, cnb.LifecycleMetada
 	}
 }
 
-func (l *LifecycleProvider) UpdateImage(cm *corev1.ConfigMap) {
+func (l *LifecycleProvider) UpdateImage(cm *corev1.ConfigMap) error {
 	lifecycle, err := l.read(context.Background(), cm)
 	if err != nil {
-		l.lifecycleData.Store(configmapRead{err: err})
```

Comment:
Do we not need to store the error case?

Reply:
Yes we do. Thanks!

Reply:
Please see 2ef676cd5739484f47a67c05e77a1341670b5dff.